src/datepicker/utils.es

```diff
@@ -16,3 +16,33 @@ export const getDate = (dateOrIso) => {
   if (dateOrIso instanceof Date) return dateOrIso;
   return parseISO(dateOrIso);
 };
+
+const stripPrefix = (key, prefix) => {
+  const strippedKey = key.substring(prefix.length);
+  return strippedKey.charAt(0).toLowerCase() + strippedKey.slice(1);
+};
+
+export const getSubTheme = (theme, prefix) => (
+  Object.entries(theme || {})
+    .reduce((acc, [key, value]) => {
```

Comment:
you seem to double check the key: 

1 .startsWith
2 .substring

just generate 1 regex with prefix and run over all, if there aint no prefix it will just do nothing

Reply:
Won't work because ...
- getSubTheme should only return the prefixed keys, not the others
- Still need to replace first char after prefix see other comment https://github.com/goodhood-eu/nebenan-form/pull/35#discussion_r412964828